lib/cisco_node_utils/syslog_settings.rb

```diff
@@ -42,6 +42,68 @@ def ==(other)
       name == other.name
     end
 
+    def console
+      console = config_get('syslog_settings', 'console')
+      if console.is_a?(Array)
+        console = console[0] == 'no' ? 'unset' : console[1]
+      end
+      console
+    end
+
+    def console=(severity)
+      if severity
+        fail ArgumentError,
+             'Console severity must be in range [0-7]' \
```

Comment:
In general we for ranges like this we just let the cli reject it.  That way if the range changes in the future we don't have to update the object.

Reply:
I can take this out, but the levels are defined by RFC 5424, so I think the likelihood of them changing in the next 5 years is low.  We're enforcing the value input on the puppet side, so it's entirely your call.

Reply:
If the enforcement is happening on the puppet side then I certainly don't think we need to do that here.  Lets keep it simple on this side.